src/components/chip-set/examples/chip-set-input.scss

```diff
@@ -1,6 +1,5 @@
 limel-chip-set[type='input'] {
-    --icon-background-color: rgb(173, 173, 173);
-    --icon-color: white;
+    --icon-background-color: rgb(var(--contrast-300));
```

Comment:
I think we should just remove the color from here, and let icons that doesn't have a style show the _actual_ default styling they would have otherwise.

Reply:
hmmm... the default is `transparent`. I thought we maybe want some more neutral icon color for the chip-set. But I don't have any strong argument to support why, except that it looked nicer (maybe because I'm so used to our current ui style?)

Reply:
:zap:'ed by moving all colours to a separate example.